### PR TITLE
[FLINK-11554][doc]Translate the community&project page of Apache Flink Web.

### PR DESCRIPTION
**Pull request by klion26**
- Changes
Translate "Community & Project Info" into Chinese

@wuchong

### CODE REVIEW

**Comment by XuQianJin-Stars** [comment]
Thanks for the effort.

Some formatting issues can refer to the Flink Translation Specifications:
https://docs.google.com/document/d/1zhGyPU4bVJ7mCIdUTsQr9lHLn4vltejhngG_SOV6g1w/edit#heading=h.by2zo7lwgzr5

**Comment by klion26** [comment]
@XuQianJin-Stars thank you for your review, I'll have a look at it.

File: community.zh.md [hunk 1]
```diff
@@ -7,31 +7,31 @@ title: "社区 & 项目信息"
 {% toc %}
 
 
-There are many ways to get help from the Apache Flink community. The [mailing lists](#mailing-lists) are the primary place where all Flink committers are present. For user support and questions use the *user mailing list*. Some committers are also monitoring [Stack Overflow](http://stackoverflow.com/questions/tagged/apache-flink). Please remember to tag your questions with the *[apache-flink](http://stackoverflow.com/questions/tagged/apache-flink)* tag. Bugs and feature requests can either be discussed on the *dev mailing list* or on [Jira]({{ site.jira }}). Those interested in contributing to Flink should check out the [contribution guide](how-to-contribute.html).
+我们可以通过多种形式从 Apache Flink 社区获得帮助。Flink commiter 主要活跃在 [邮件列表](#mailing-lists)。对于用户支持和问题咨询，则可以通过 *用户邮件列表* 获得帮助。有些 Committer 同时会关注 [Stack Overflow](http://stackoverflow.com/questions/tagged/apache-flink)。请在提问的时候记得添加 apache-flink 的标签。问题反馈以及新特性的讨论则可以在 *开发邮件列表* 或者 [Jira]({{ site.jira }}) 上进行讨论。有兴趣对 Flink 进行贡献的人请查阅 [贡献指南](how-to-contribute.html)。
```
Comment (XuQianJin-Stars) [inline community.zh.md]: 形式 -> 方式 ?   
[邮件列表](#mailing-lists)  -> [邮件列表]({{ site.baseurl }}/zh/community.html#mailing-lists)
[贡献指南](how-to-contribute.html) -> [贡献指南]({{ site.baseurl }}/zh/how-to-contribute.html)

File: community.zh.md [hunk 2]
```diff
@@ -7,31 +7,31 @@ title: "社区 & 项目信息"
...
+如果你的邮件中包含代码，请根据如下要求进行检查：
 
-1. you do not link to files in external services as such files can change, get deleted or the link might break and thus make an archived email thread useless
-2. you paste text instead of screenshots of text
-3. you keep formatting when pasting code in order to keep the code readable
-4. there are enough import statements to avoid ambiguities
+1. 请确认你所使用的外链不会被修改、删除等，这些操作会使得归档后的邮件变得无效
+2. 粘贴文字而不是文字的截图
+3. 对代码进行格式化操作，以提高可读性
+4. 添加足够的上下文，确保代码没有模棱两可的地方
```
Comment (XuQianJin-Stars) [inline community.zh.md]: Remove the empty line after `1.`

File: community.zh.md (hunk 1 above)
Comment (XuQianJin-Stars) [reply community.zh.md]: {{site. Baseurl}} -> {{site. Baseurl}}/zh covers all the following

File: community.zh.md [hunk 3]
```diff
@@ -413,17 +412,17 @@ In 2017, Flink Forward came to San Francisco to welcome the Apache Flink communi
 
 </table>
 
-You can reach committers directly at `<apache-id>@apache.org`. A list of all contributors can be found [here]({{ site.FLINK_CONTRIBUTORS_URL }}).
+可以通过 `<apache-id>@apache.org` 直接联系 commiter。可以在所有的 [这里]({{ site.FLINK_CONTRIBUTORS_URL }}) 找到所有的贡献者。
```
Comment (XuQianJin-Stars) [inline community.zh.md]: 可以在所有的 [这里]({{ site.FLINK_CONTRIBUTORS_URL }}) 找到所有的贡献者。 -> 可以在 [这里]({{ site.FLINK_CONTRIBUTORS_URL }}) 找到所有的贡献者。

File: community.zh.md [hunk 4]
```diff
@@ -465,8 +464,8 @@ The following people were very kind to mentor the project while in incubation.
   </tr>
 </table>
 
-# Materials / Apache Flink Logos
+# 其他资料以及 Apache Flink 徽标
 
-The [materials page]({{ site.baseurl }}/material.html) offers assets such as the Apache Flink logo in different image formats, or the Flink color scheme.
+[资料页面]({{ site.baseurl }}/material.html) 提供了不同尺寸以及不同颜色的 Apache Flink 徽标
```
Comment (XuQianJin-Stars) [inline community.zh.md]: {{site. Baseurl}} -> {{site. Baseurl}}/zh

File: community.zh.md (hunk 1 above)
Comment (wuchong) [reply community.zh.md]: The relative link of "贡献指南" is fine. We only need to adapt absolute links.

File: community.zh.md [hunk 5]
```diff
@@ -7,31 +7,31 @@ title: "社区 & 项目信息"
...
+    <th class="text-center">摘要</th>
+    <th class="text-center">退订</th>
+    <th class="text-center">发送</th>
+    <th class="text-center">归档</th>
   </thead>
   <tr>
     <td>
       <strong>news</strong>@flink.apache.org<br>
-      <small>News and announcements from the Flink community</small>
+      <small>Flink 社区的新闻和公告</small>
```
Comment (wuchong) [inline community.zh.md]: Translate "Subscribe" , "Unsubscribe", "Read only list", "Archives" in the table?

File: community.zh.md [hunk 6]
```diff
@@ -121,70 +121,69 @@ If you send us an email with a code snippet, make sure that:
   </tr>
 </table>
 
-<b style="color:red">Please make sure you are subscribed to the mailing list you are posting to!</b> If you are not subscribed to the mailing list, your message will either be rejected (dev@ list) or you won't receive the response (user@ list).
+<b style="color:red">请确认你订阅了你所发送到的邮件列表!</b> 如果你没有订阅对应的邮件列表的话，对于 dev 邮件列表，你的消息会被拒收，对于 user 邮件列表，你则收不到相应的回复。
```
Comment (wuchong) [inline community.zh.md]: 请确认你订阅了你所发送到的邮件列表！ ->
在给邮件列表发邮件之前，请确认已经订阅了该邮件列表！

File: community.zh.md [hunk 7]
```diff
@@ -121,70 +121,69 @@ If you send us an email with a code snippet, make sure that:
...
+请确认你为自己的问题打上了正确的标签，从而可以获得 Flink 社区的帮助。
 
-## Issue Tracker
+## Issue 追踪
 
-We use Jira to track all code related issues: [{{ site.jira }}]({{ site.jira }}).
+我们使用 Jira 进行所有代码相关的 issues 追踪  [{{ site.jira }}]({{ site.jira }})。
 
-All issue activity is also mirrored to the issues mailing list.
+所有 issue 的活动也会同步到 issue 对应的邮件列表。
```
Comment (wuchong) [inline community.zh.md]: issue 对应的邮件列表 -> issues 邮件列表   ?
这里应该是特指 issues@flink.apache.org 吧？

File: community.zh.md [hunk 8]
```diff
@@ -121,70 +121,69 @@ If you send us an email with a code snippet, make sure that:
...
 ## Meetups
 
-There are plenty of meetups on [meetup.com](http://www.meetup.com/topics/apache-flink/) featuring Flink.
+在 [meetup.com](http://www.meetup.com/topics/apache-flink/) 上可以找到很多 Flink 相关的 meetup。
 
-## Source Code
+## 源代码
 
-### Main source repositories
+### 源代码仓库地址
```
Comment (wuchong) [inline community.zh.md]: 仓库地址 -> 主仓库

File: community.zh.md [hunk 9]
```diff
@@ -121,70 +121,69 @@ If you send us an email with a code snippet, make sure that:
...
-### Flink-shaded repositories (shaded dependency libraries)
+### Flink-shaded 仓库 (shaded 的依赖包)
 
-- **ASF writable**: [https://gitbox.apache.org/repos/asf/flink-shaded.git](https://gitbox.apache.org/repos/asf/flink-shaded.git)
-- **GitHub mirror**:  [https://github.com/apache/flink-shaded.git](https://github.com/apache/flink-shaded.git)
+- **ASF 仓库**: [https://gitbox.apache.org/repos/asf/flink-shaded.git](https://gitbox.apache.org/repos/asf/flink-shaded.git)
+- **GitHub 镜像**:  [https://github.com/apache/flink-shaded.git](https://github.com/apache/flink-shaded.git)
 
-### Flink Website repositories
+### Flink 网站代码库
```
Comment (wuchong) [inline community.zh.md]: 代码库 -> 仓库

翻译保持一致

File: community.zh.md [hunk 10]
```diff
@@ -121,70 +121,69 @@ If you send us an email with a code snippet, make sure that:
...
-- **GitHub mirror**:  [https://github.com/apache/bahir-flink.git](https://github.com/apache/bahir-flink.git)
+- **ASF 仓库**: [https://git-wip-us.apache.org/repos/asf/bahir-flink.git](https://git-wip-us.apache.org/repos/asf/bahir-flink.git)
+- **GitHub 镜像**:  [https://github.com/apache/bahir-flink.git](https://github.com/apache/bahir-flink.git)
 
 
-## Training
+## 培训
 
-[dataArtisans](http://data-artisans.com) currently maintains free Apache Flink training. Their [training website](http://training.data-artisans.com/) has slides and exercises with solutions. The slides are also available on [SlideShare](http://www.slideshare.net/dataArtisans/presentations).
+[ververica](https://www.ververica.com/) 目前维护了一些免费的 Apache Flink 培训。[培训页面](https://training.ververica.com/) 有相应的演示文档和带解答的练习。当然还可以在 [SlideShare](http://www.slideshare.net/dataArtisans/presentations) 找到所有的演示文档。
```
Comment (wuchong) [inline community.zh.md]: nice change!

建议 ververica -> Ververica

File: community.zh.md [hunk 11]
```diff
@@ -121,70 +121,69 @@ If you send us an email with a code snippet, make sure that:
...
-## Project Wiki
-
-The Apache Flink <a href="https://cwiki.apache.org/confluence/display/FLINK/Apache+Flink+Home" target="_blank">project wiki</a> contains a range of relevant resources for Flink users. However, some content on the wiki might be out-of-date. When in doubt, please refer to the <a href="{{ site.docs-stable }}" target="_blank">Flink documentation</a>.
+## 项目 Wiki
+Apache Flink 的 <a href="https://cwiki.apache.org/confluence/display/FLINK/Apache+Flink+Home" target="_blank">项目 wiki</a> 包含了大量的 Flink 用户相关的资源。不过 wiki 上的资料可能会过时，如果有任何疑问，请查阅 <a href="{{ site.docs-stable }}" target="_blank">Flink 文档</a>
 
 ## Flink Forward
 
-Flink Forward 2015 (October 12-13, 2015) was the first conference to bring together the Apache Flink developer and user community. You can find [slides and videos](http://2015.flink-forward.org/?post_type=session) of all talks on the Flink Forward 2015 page.
+Flink Forward 2015 (2015 年 10 月 12-13) 是第一届把 Apache Flink 开发者和用户聚集起来的大会。可以在 [演示文档以及视频](http://2015.flink-forward.org/?post_type=session) 页面中找到 2015 年 Flink Forward 的所有资料
```
Comment (wuchong) [inline community.zh.md]: 2015 年 10 月 12-13  末尾加个 "日"，下文类似。 更符合中文语感。

File: community.zh.md [hunk 12]
```diff
@@ -413,17 +412,17 @@ In 2017, Flink Forward came to San Francisco to welcome the Apache Flink communi
...
 </table>
 
-You can reach committers directly at `<apache-id>@apache.org`. A list of all contributors can be found [here]({{ site.FLINK_CONTRIBUTORS_URL }}).
+可以通过 `<apache-id>@apache.org` 直接联系 commiter。可以在所有的 [这里]({{ site.FLINK_CONTRIBUTORS_URL }}) 找到所有的贡献者。
 
-## Former mentors
+## 前导师们
 
-The following people were very kind to mentor the project while in incubation.
+下面这些人在项目孵化的时候给了非常好的引导作用。
```
Comment (wuchong) [inline community.zh.md]: 给了 -> 起到了

File: community.zh.md (hunk 4 above)
Comment (wuchong) [reply community.zh.md]: 末尾句号。

File: community.zh.md (hunk 7 above)
Comment (klion26) [reply community.zh.md]: Yes, it is `issues@flink.apache.org`

File: community.zh.md [hunk 13]
```diff
@@ -7,184 +7,183 @@ title: "社区 & 项目信息"
...
-    <td class="text-center"><i class="fa fa-pencil-square-o"></i> <a href="mailto:user@flink.apache.org">Post</a></td>
+    <td class="text-center"><i class="fa fa-pencil-square-o"></i> <a href="mailto:user-subscribe@flink.apache.org">订阅</a></td>
+    <td class="text-center"><i class="fa fa-pencil-square-o"></i> <a href="mailto:user-digest-subscribe@flink.apache.org">订阅</a></td>
+    <td class="text-center"><i class="fa fa-pencil-square-o"></i> <a href="mailto:user-unsubscribe@flink.apache.org">退订</a></td>
+    <td class="text-center"><i class="fa fa-pencil-square-o"></i> <a href="mailto:user@flink.apache.org">发送</a></td>
     <td class="text-center">
-      <a href="http://mail-archives.apache.org/mod_mbox/flink-user/">Archives</a> <br>
-      <a href="http://apache-flink-user-mailing-list-archive.2336050.n4.nabble.com/">Nabble Archive</a>
+      <a href="http://mail-archives.apache.org/mod_mbox/flink-user/">归档</a> <br>
+      <a href="http://apache-flink-user-mailing-list-archive.2336050.n4.nabble.com/">手动归档</a>
```
Comment (klion26) [inline community.zh.md]: 手动归档这个暂时存疑，如果有更好的翻译欢迎提出

**Comment by klion26** [comment]
@XuQianJin-Stars @wuchong thank you for your review, I've updated the translation, please have a look when you are free.

File: community.zh.md (hunk 13 above)
Comment (wuchong) [reply community.zh.md]: Nabble 不用翻译。 我觉得 "Nabble 归档" 就行

**Comment by wuchong** [comment]
Thanks @klion26 for the effort. Overall looks good to me.

Will merge this. 

Thanks,
Jark

**Pull request closed without merging**